clients/CMakeLists.txt

```diff
@@ -1,5 +1,23 @@
 # ########################################################################
-# Copyright 2016-2022 Advanced Micro Devices, Inc.
+# Copyright 2016-2022 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Doesn't the template use `Copyright (C)`?

Reply:
It does... I just noticed that clang-format is removing it. Is that intentional?

Reply:
Sorry, not clang format. The perl snippet in the pre-commit script that manages copyright dates didn't have a (C) in the replacement string so it was disappearing during commit.

`/usr/bin/perl -pi -e 'INIT { exit 1 if !-f $ARGV[0] || -B $ARGV[0]; $year = (localtime)[5] + 1900 }
            s/^([*\/#[:space:]]*)Copyright\s+(?:\(C\)\s*)?(\d+)(?:\s*-\s*\d+)?/qq($1Copyright $2@{[$year != $2 ? "-$year" : ""]})/ie
            if $. < 10' "$file" && git add -u "$file"`

Fixed so it won't do that anymore.

`/usr/bin/perl -pi -e 'INIT { exit 1 if !-f $ARGV[0] || -B $ARGV[0]; $year = (localtime)[5] + 1900 }
            s/^([*\/#[:space:]]*)Copyright\s+(?:\(C\)\s*)?(\d+)(?:\s*-\s*\d+)?/qq($1Copyright \(C\) $2@{[$year != $2 ? "-$year" : ""]})/ie
            if $. < 10' "$file" && git add -u "$file"`

Reply:
Correct. Updating that script is a necessary part of this change.